src/titanium-company-selector.ts

```diff
@@ -179,28 +191,39 @@ export class TitaniumCompanySelectorElement extends PolymerElement {
   }
 
   svg {
-    fill: var(--app-text-color-lighter);
+    fill: var(--titanium-input-color, #2c7eea);
     width: 24px;
     height: 24px;
   }
 
-  dual-ring-spinner {
+  lds-ring {
     display: inline-block;
-    width: 26px;
-    height: 26px;
+    position: relative;
+    width: 24px;
+    height: 24px;
   }
-  dual-ring-spinner:after {
-    content: " ";
+  lds-ring div {
+    box-sizing: border-box;
     display: block;
-    width: 18px;
-    height: 18px;
-    margin: 1px;
+    position: absolute;
+    width: 21px;
+    height: 21px;
+    margin: 3px;
+    border: 3px solid var(--titanium-input-color, #2c7eea);
     border-radius: 50%;
-    border: 3px solid var(--material-primary-color);
-    border-color: var(--material-primary-color) transparent var(--material-primary-color) transparent;
-    animation: dual-ring-spinner 1.2s linear infinite;
+    animation: lds-ring 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+    border-color: var(--titanium-input-color, #2c7eea) transparent transparent transparent;
+  }
+  lds-ring div:nth-child(1) {
```

Comment:
dangerous selector `lds-ring div` because web components polyfill uses divs for scoping.  is a   use `lds-ring>div`